### PR TITLE
Add CC License

### PR DESCRIPTION
**Pull request by titaniumbones**
Mozilla is asking for a license here (for global sprint), and we should probably have one in any case.  I put GPL 3 in htis PR but since this repo is mostly documentation we might want LGPL or a CC license, any thoughts?

### CODE REVIEW

**Comment by Mr0grog** [comment]
Ha, I hadn’t even noticed (or thought about) the fact that this repo didn’t have a license. I don’t think LGPL vs. GPL would be a meaningful difference for this repo, so if the question were between just those, I’d say just stick with GPL. It *does* seem like, since this is pure documentation, a CC license (Attribution-ShareAlike 4.0 International?) might be more appropriate. I don’t have expertise or strong opinions here, though.

We should also add a license section to the README, as in the other repos, e.g. https://github.com/edgi-govdata-archiving/web-monitoring-db#license--copyright

**Comment by dcwalk** [comment]
yeah, I think we go for CC in this case to license the docs...? (and it seems like that is all they are looking for: https://github.com/mozilla/global-sprint/issues/102#issuecomment-304902471

Just wrapping up other things and can circle back on this in an hour or so.

**Comment by dcwalk** [comment]
Updated to a CC license & included license block in readme, able to review?

File: README.md [hunk 1]
```diff
@@ -23,6 +23,11 @@ This repository is for EDGI [Web Monitoring Project](https://github.com/edgi-gov
 
 EDGI is already monitoring tens of thousands of pages and will eventually be monitoring tens of millions (or even as many as ~1 billion). Currently there is a lot of manual labour that goes into reviewing all changes, regardless of whether they are meaningful or not. Any system will need to emphasize usability of the UI and efficiency of computational resources.
 
+- [Project Goals](#project-goals)
+- [How to Help](#how-to-help)
+- [Project Overview](#project-overview)
+- [License & Copyright](#license-copyright)
```
Comment (Mr0grog) [inline README.md]: This link isn’t working—I think it needs to be `#license--copyright` (double dash because of the `&` in the title).

**Comment by Mr0grog** [comment]
Oh, actually, should we include the CC license logo in the README?

<a rel="license" href="http://creativecommons.org/licenses/by-sa/4.0/">Creative Commons License</a>

**Comment by Mr0grog** [comment]
(Went ahead and just fixed the link)

**Comment by dcwalk** [comment]
friendly commits welcome! -- added in the image, but went for the compact one because it kept looking weird on the line.

Merging in b/c timing

**Pull request merged**